tests/foreman/ui/test_activationkey.py

```diff
@@ -55,14 +55,14 @@ def setUpClass(cls):
         super(ActivationKey, cls).setUpClass()
 
     def create_sync_custom_repo(self, product_name=None, repo_name=None,
-                                repo_url=None, repo_type=None):
+                                repo_url=None, repo_type=None, org_id=None):
         """Create product/repo, sync it and returns repo_id"""
         product_name = product_name or gen_string("alpha", 8)
         repo_name = repo_name or gen_string("alpha", 8)
         # Creates new product and repository via API's
         product_attrs = entities.Product(
             name=product_name,
-            organization=self.org_id
+            organization=org_id or self.org_id
```

Comment:
Or what we can do is, handle this issue for once inside `fetch_rhproduct` and `fetch_repoid`, so there is no need to convert the type every time we call these methods.
```
prd_id = entities.Product().fetch_rhproduct_id(name=product, org_id=org_id)
 repo_id =  entities.Repository().fetch_repoid(name=repo, org_id=org_id)
```